src/main/java/com/gregtechceu/gtceu/data/recipe/configurable/RecipeRemoval.java

```diff
@@ -266,7 +296,17 @@ private static void flintAndSteelRequireSteel(Consumer<ResourceLocation> registr
         registry.accept(new ResourceLocation("minecraft:flint_and_steel"));
     }
 
+    /**
+     * Removes the vanilla recipe for an item that would has BOTH a normal recipe as well as a GT recipe in normal
```

Comment:
Update to be more clear

"This removes the vanilla recipe for an item that would have both a normal and a GT recipe in normal recipe mode."